### ISSUE TITLE
Zeth'kur logs not showing up

### ISSUE THREAD

**Issue by 08freaxy** [Geigerkind/Legacy-Logs-Bugtracker#91]
Since around 16th of may the raid logs for zeth'kur server aren't shwoing up. They do upload, but they simply won't show in the raid section.
Maybe this issue has something to do with the SQL error being fixed for the zeth'kur armory pages, as this was around the same time fixed.

**Comment by RussianE39** [comment]
Yes, it's definitely related to ZK SQL error fix, because the only time I was able to upload log, when realm showed as "Unknown" after processing.

**Comment by RussianE39** [comment]
1. Logs from ZK could be uploaded by Java Launcher and they will appear as "Unknown" realm in a list
2. We have char in guild, named "Onyxia", and parser probably cuts out everything, related to him. I've checked presence of information in DPSMate log for him, but absense of all the information on web page after raid log parsing.

**Comment by 08freaxy** [comment]
I'll try point 1 tonight.
Point 2 might be cause by the dpsmate bug where it doesn't show yourself. This has been fixed in one of the latest patches. I'd suggest for your logger to download the new dpsmate.

**Comment by RussianE39** [comment]
Point 2 - he is present in DPSMate log file, and all of our loggers using either 114 build or 120.

**Comment by Geigerkind** [comment]
Will take care of it!

**Comment by Geigerkind** [comment]
Fixed!

**Comment by RussianE39** [comment]
We can now use both loaders? what about point no.2 ?

**Comment by Geigerkind** [comment]
Hmm will see
